frontend/packages/pipelines-plugin/src/components/pipelineruns/list-page/PipelineRunRow.tsx

```diff
@@ -85,15 +87,20 @@ const PipelineRunRow: React.FC<RowFunctionArgs<PipelineRunKind>> = ({ obj, custo
         <PipelineRunVulnerabilities pipelineRun={obj} condensed />
       </TableData>
       <TableData className={tableColumnClasses.status}>
-        <PLRStatus obj={obj} taskRuns={PLRTaskRuns} />
+        <PLRStatus obj={obj} taskRuns={PLRTaskRuns} iconOnly />
```

Comment:
On wide screens/windows I would expect still the label:

Can we hide the labels maybe just on sm-md and show it on lg+?

![image](https://github.com/openshift/console/assets/139310/cb859871-7a16-4c00-842d-6c45ebe663a1)